### ISSUE TITLE
[Accessibility] Switch component is not accessible

### ISSUE THREAD

**Issue by andresgalante** [auth0/cosmos#1294]
There are some things to do to make the switch component accessible:

- Add a `for`on the label
- Add an `id` on the input
- Add a `aria-label="..."` on the input
- Add a `checked` on the active input
- Add a `disabled` on the disabled input
